### ISSUE TITLE
Enhancement request : Manual value for minimum contract

### ISSUE THREAD

**Issue by Mglt-b** [hunterjm/futgui#92]
Hello,

Can you add possibility for the minimum of contract required for the bid ?

Thank you

futgui/core/bid.py
 # Must have contract
                    if item['contract'] < 1:
                        continue

It's actually 1 minimum contract ?

**Comment by hunterjm** [comment]
Yes, the bot won't buy expired players. I see no real reason to have configuration past that though...
